templates/csharp/command-and-response/appsettings.Development.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "AllowedHosts": "*",
-  "BOT_ID": "$botId$",
-  "BOT_PASSWORD": "$bot-password$"
+  "BOT_ID": "",
+  "BOT_PASSWORD": "",
+  "BOT_TYPE": ""
```

Comment:
It seems some templates (this one) change *appsettings.Development.json* but some (ai-bot) not. Expected?

Reply:
Missed. Let me update.